ME-MYSELF-I/Weekly-meetings.md

```diff
@@ -1,5 +1,48 @@
 # Weekly Update
 
+* [22nd September 2017](#date-22nd-september-2017)
+
+### Date: 22nd September 2017
+
+#### Who did I help this week?
+
+* I helped Xavier run a matlab script within a python workflow.
+* I helped Todd with some heudiconv debugging
+* I helped a nipype contributor with their PR.
+
+#### Who helped me this week?
+
+* Isabelle helped by curating the BANDA NIH toolbox metadata
+
+#### What did I achieve?
+
+* I answered questions on neurostars, and reviewed a nipype PR.
+* I finished running BANDA fully through the HCP BIDS-App
+* I continued refactoring heudiconv into a package
+* I created some figures to show at the data blitz
+
+#### What did I struggle with?
+
+* adding brainmask to the mindboggle123 pipeline
+
+#### What would I like to work on next week?
+
+* Finish Packaging heudiconv
+* Complete preprocessing for voice (mindboggle + fmriprep)
+* Finally convert MTBI-Texas to BIDS(again) and validate
+
+#### Where do I need help?
+
+* Mindboggle with lesions (Satra)
+* Finding more controls for voice
```

Comment:
can you list: `age, sex` that you still need?

Reply:
I updated the issue #1 with a table